README.md

```diff
@@ -1 +1,57 @@
-# microshift
+# Microshift
+Microshift (aka "µShift") is OpenShift<sup>1</sup> Kubernetes in a small form factor and optimized for edge computing.
```

Comment:
We dropped all uShift usage from the code, but I'd argue that it's a µShift is the brand, and so should be used in documentation instead of "microshift", except for `µShift (aka "Microshift")`up front.

_OR_. we should not use µShift in docs at all if we only ever mention the branding spelling once.

Reply:
And to pile on - we can't use `µShift` throughout the docs without it resulting in confusion when running `$ microshift`

Reply:
Good points. Will remove µShift from text. Probably more something for a future logo :)